### PR TITLE
SCALRCORE-15619 HCL > Rewrite pyhcl by using Go parser as C library

### PR DESCRIPTION
**Pull request by lbn**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+steps:
+  - name: fetch
+    image: alpine/git
+    commands:
+      - git fetch --tags
+    when:
+      event:
+        - tag
+  - name: build
+    image: larcher/manylinux-golang
```
Comment (maratkomarov) [inline .drone.yml:14]: @lbn leftover: please re-push image to `scalrlabs` account (creds are in confluence), and include it's Dockerfile into the repo.

File: pygohcl.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/hcl/v2/hclparse"
+)
+
+//export Parse
+func Parse(a *C.char) C.parseResponse {
+	input := C.GoString(a)
+	hclFile, diags := hclparse.NewParser().ParseHCL([]byte(input), "tmp.hcl")
```
Comment (maratkomarov) [inline pygohcl.go:19]: @lbn leftover: usually we deal with files, but the only interface `pygohcl` has is `.loads(...)`, so the original filename disappears here, and Go layer assigns a placeholder name `tmp.hcl` to the payload, and HCLParseError confusedly carries it

```
HCLParseError: invalid HCL: tmp.hcl:1,1-2: Invalid character; Single quotes are not valid. Use double quotes (") to enclose strings., tmp.hcl:1,1-2: Argument or block definition required; An argument or block definition is required here.
```

File: hcl_to_json.go [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+	if err != nil {
+		return err
+	}
+
+	for _, label := range block.Labels {
+		if inner, exists := out[key]; exists {
+			var ok bool
+			out, ok = inner.(jsonObj)
+			if !ok {
+				// TODO: better diagnostics
```
Comment (maratkomarov) [inline hcl_to_json.go:69]: @lbn is this our own issue, or borrowed from original tmccombs/hcl2json?

Comment (lbn) [reply hcl_to_json.go:69]: Copy and pasted.

**Pull request merged**